src/main/java/com/neprozorro/rest/LotInfoResource.java

```diff
@@ -0,0 +1,34 @@
+package com.neprozorro.rest;
+
+import com.neprozorro.model.LotInfo;
+import com.neprozorro.rest.dto.LotInfoCriteriaRequestDto;
+import com.neprozorro.rest.dto.LotInfoMapper;
+import com.neprozorro.rest.dto.LotInfoResponseDTO;
+import com.neprozorro.service.LotInfoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/lot-info")
+@RequiredArgsConstructor
+public class LotInfoResource {
+
+    private final LotInfoService lotInfoService;
+    private final LotInfoMapper lotInfoMapper;
+
+    @GetMapping
+    public List<LotInfo> findAll() {
+        return lotInfoService.findAll();
+    }
```

Comment:
Maybe we need ResponseStatus?

Reply:
I agree. However, we don't do this anywhere. That's why I decide, to leave it to refactoring and/or optimization when we decide how exactly we will do it. Let's discuss it at the meeting.

Reply:
Done

Reply:
I can't see that it's done, is it supposed to be annotation @ResponseStatus?